bundle.yaml.j2

```diff
@@ -50,14 +50,14 @@ applications:
   traefik-admin:
     charm: traefik-k8s
     revision: 164
-    channel: latest/edge
+    channel: 1.0/stable
```

Comment:
I'm afraid we can't pin that revision. It doesn't support forward-auth and will break the integration with oathkeeper (requires traefik rev166+)